src/swarm-scenario/Swarm/Game/World/Syntax.hs

```diff
@@ -83,7 +72,7 @@ data Axis = X | Y
 instance PrettyPrec Axis where
   prettyPrec _ = \case X -> "x"; Y -> "y"
 
-data Op = Not | Neg | And | Or | Add | Sub | Mul | Div | Mod | Eq | Neq | Lt | Leq | Gt | Geq | If | Perlin | Reflect Axis | Rot Rot | Mask | Overlay | Abs
+data Op = Not | Neg | And | Or | Add | Sub | Mul | Div | Mod | Eq | Neq | Lt | Leq | Gt | Geq | If | Perlin | Mask | Overlay | Abs | IMap
```

Comment:
Were there no worlds using reflection and rotation?

Reply:
Correct.  Reflection wasn't even fully implemented, as it turns out.  And rotation is not very interesting until you have things like structures you can refer to from within the language.